kpconv_torch/datasets/S3DIS.py

```diff
@@ -865,20 +865,26 @@ def load_evaluation_points(self, file_path):
     def read_input(self, filename):
         """Read all the input files that belong to the dataset
 
-        Ply files are read by training and testing commands.
+        PLY files are read by training and testing commands.
         """
-        file_extension = filename.split(".")[-1]
-        if file_extension == "ply":
+        file_extension = Path(filename).suffix
+        if file_extension == ".ply":
             # Read ply file
             data = read_ply(filename)
             points = np.vstack((data["x"], data["y"], data["z"])).T
             colors = np.vstack((data["red"], data["green"], data["blue"])).T
-            labels = data["classification"]
```

Comment:
Comes from the rebase on the master but generates an error

Reply:
There is no "original .ply", we produce them through preprocessing. You should drop your preprocessing output and regen it.

Reply:
ok